tasks.go

```diff
@@ -56,6 +62,24 @@ func GetAllTasks(basepath string) ([]Task, error) {
 	return tasks, nil
 }
 
+// GetNagiosDataTask will return an array of Tasks each of which will dump the nagios data for one project.
+func GetNagiosTasks(projects []string, basepath string) ([]Task, error) {
+	var tasks []Task
+	for _, p := range projects {
```

Comment:
Here is the place where instead of creating as many tasks as projects, I'd find the 1 Nagios we are looking for, and then create 1 task.